Lib/urllib/parse.py

```diff
@@ -383,6 +383,10 @@ def urlparse(url, scheme='', allow_fragments=True):
 
     Note that % escapes are not expanded.
     """
+    if not isinstance(url, (str, bytes, bytearray)):
+        raise TypeError(f'Expected string or bytes: got {type(url)}')
+    if not isinstance(scheme, (str, bytes, bytearray)):
+        raise TypeError(f'Expected string or bytes: got {type(scheme)}')
```

Comment:
Why dint you keep the wording to `a string or bytes`  instead of `string or bytes` as in the news entry? Sounds better to me. Maybe others have a better view. cc @pablogsal who likes making messages better.

Reply:
Because I wrote the news entry second! :-D

I'll push a commit to change it to `Expected a string or bytes object: ...`

Reply:
I like @nanjekyejoannah suggestion :)